EIPS/eip-867.md

```diff
@@ -0,0 +1,177 @@
+---
+eip: 867
+title: Standardized Ethereum Recovery Proposals
+author: Dan Phifer (dp@musiconomi.com), James Levy (james@taptrust.com), Reuben Youngblom (reuben@taptrust.com)
+type: Meta
+status: Draft
+created: 2018-02-02
+replaces: 156
```

Comment:
Apparently, this causes tests to fail, so since EIP-156 was never merged, you should remove it.

```
- ./_site/EIPS/eip-867.html
  *  internally linking to eip-156, which does not exist (line 55)
     <a href="eip-156">156</a>
htmlproofer 3.8.0 | Error:  HTML-Proofer found 1 failure!
```